### ISSUE TITLE
Integrate finetune+weights into main.py call post evaluation

### ISSUE THREAD

**Issue by uchuuronin** [riakashyap/646_Project#71]
At this point, you can remove all evaluation scripts (**eval_reranker.py**, which tests the fine-tune model and different loss functions, and **eval_weights.py**, which experiments with different weight applications) and update the main.py model_path file to call the reranker and add weights there directly.

Do only once both finetune and weight functions are fully tested and merged
